### DISCUSSION TITLE
bitcoin testnet support integration

### DISCUSSION THREAD

**Discussion by ansigroup** [phantom/docs#144] [Help & Support] [answered]
### Summary

Hello,
please add bitcoin testnet support to wallet

### Steps to Reproduce

Developer Settings: no bitcoin testnet

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@ansigroup Thank you for providing this feature request! Please also consider posting this to the [Phantom Feature Request Board](https://phantom.canny.io/admin/feedback/feature-requests/p/portfolio-watchlist-views) and asking any others interested in this feature to upvote it.

We are very excited to extend our bitcoin support!
